app/views/users/change_email.html.erb

```diff
@@ -19,10 +19,8 @@
     <dt><%= label_tag :email_confirmation, ts("Confirm New Email") %></dt>
     <dd><%= email_field_tag :email_confirmation %></dd>
 
-    <% unless @user.recently_reset? %>

```

Comment:
This is just a note for us that this changes some behavior and fixes https://otwarchive.atlassian.net/browse/AO3-3717